components/hab/src/cli.rs

```diff
@@ -75,6 +75,22 @@ pub fn get() -> App<'static, 'static> {
                     "Ring key name, which will encrypt communication messages")
             )
         )
+        (@subcommand job =>
+            (about: "Commands relating to job control")
```

Comment:
Might be good to make this text "build job control" so that it gives a bit more indication that these commands are for builds.